priv/templates/jsonapi/controller_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule <%= module %>ControllerTest do
```

Comment:
we should prolly make sure this follows the req response codes from http://jsonapi.org/format/